tests/update_script.prm

```diff
@@ -17,7 +17,7 @@ subsection Adiabatic conditions model
   subsection Compute profile
     set Use surface condition function = true
     subsection Surface condition function
-      set Function expression = 1e-2*t; 1613.0+1e-13*t
+      set Function expression = 1e-2/(365.2425*24*60*60)*t; 1613.0+1e-13/(365.2425*24*60*60)*t
```

Comment:
if `t` is now set to `t/year_in_seconds` but you want the function expression to remain unchanged, dont you have to set it to `1e-2 * year_in_seconds * t'`, where `t' = t/year_in_seconds`?

Reply:
Yes, you are right... I changed them.

Reply:
It passed the tests!